extensions/braze/actions/sendEmail/config/fields.ts

```diff
@@ -0,0 +1,33 @@
+import { type Field, FieldType } from '@awell-health/extensions-core'
+import { type ZodTypeAny, z } from 'zod'
+
+export const fields = {
```

Comment:
@danijel How/where is the recipient defined?

Reply:
TBH, I don't think you define recipient, I think you define audience conditions and that it sends messages to all the people who satisfy these conditions, so I think we'll need a json audience field, or something similar where we build json our selves. API example uses audience, here is full example https://www.braze.com/docs/api/endpoints/messaging/send_messages/post_send_messages/#example-request